pkg/vmprovider/providers/vsphere/vmprovider_vm_test.go

```diff
@@ -352,254 +370,152 @@ func vmTests() {
 					Expect(ethDevice.MacAddress).NotTo(Equal(ethCard.MacAddress))
 
 					p := devList.SelectByType(&types.VirtualPCIPassthrough{})
-					Expect(p).To(HaveLen(1))
+					Expect(p).To(HaveLen(2))
 					pciDev1 := p[0].GetVirtualDevice()
 					pciBacking, ok1 := pciDev1.Backing.(*types.VirtualPCIPassthroughVmiopBackingInfo)
```

Comment:
Nit: `pciBacking` -> `pciBacking1` here, and `pciBacking1` -> `pciBacking2` below to keep the number suffix consistent. Or scope these in separate blocks.

Reply:
That was the original code, I just moved it around. But sure, can take this.